manual/src/refman/modtypes.etex

```diff
@@ -262,6 +262,18 @@ No restrictions are placed on the type of the functor argument; in
 particular, a functor may take another functor as argument
 (``higher-order'' functor).
 
+One can also write
+\begin{center}
+@'functor' '(' name_1 ':' module-type_1 ')' \ldots
+           '(' name_n ':' module-type_n ')' '->' module-type@
+\end{center}
+instead of
+\begin{center}
+@'functor' '(' name_1 ':' module-type_1 ')' '->' \ldots '->'
+ 'functor' '(' name_n ':' module-type_n ')' '->' module-type@
+\end{center}
+to simplify the type expression for ``higher-order'' functor.
```

Comment:
"higher-order" functor normally refers to a functor with an argument that is itself a functor, which is not what is being described here. I would suggest something along the following lines instead:
```
When the result module type is itself a functor, one may write
< shorthand form>
instead of
< long form >
```